cloud-config/README.md

```diff
@@ -1,6 +1,135 @@
+    
+    View them locally with --dry-run along with other info.  Does not deploy
+    $ bin/deploy --dry-run
+
+    Add options like --test, --release-candidate, or --candidate to see the differences in run vars.
+    The ROLE should change along with other variables like ENCD_INDEX_PRIMARY, ENCD_INDEX_VIS,
+    ENCD_INDEX_REGION.  The env vars are prefixed with ENCD_ as to not conflict with other env vars.
+
+## Live Deployments
+    Here we'll deploy some instances using the --full-build args to avoid needing amis.  Building
+    from scratch is easier for cloud config development.
+
+#### QA/Development demo: app-es-pg-template.yml
+    $ bin/deploy --full-build
+
+    ### Output
+    Deploying app-es-pg
+    $ bin/deploy --full-build
+    create instance and wait for running state
+    ####
+
+#### Demo Cluster(es-wait): es-nodes-template.yml and app-pg-template
+    $ export CLUSTER_NAME='encd-dev'
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-wait
+   
+    ### Output
+    Deploying es-nodes
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-wait
+    Create instance and wait for running state
+    # Deploying Head ES Node(172.31.26.236): encd-dev-datamaster
+    ###
+
+    The IP address is the --es-ip used to deploy a frontend.
+    $ export ES_IP='172.31.26.236'
+
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-ip "$ES_IP"
+    
+    ### Output
+    Deploying app-pg
+    $ bin/deploy --full-build --cluster-name encd-dev --es-ip 172.31.26.236
+    Create instance and wait for running state
+
+    Deploying Frontend(172.31.23.80): https://encd-dev.demo.encodedcc.org
+    ###
+
+
+#### Demo with postgres pointing at Demo Cluster: app-pg-template.yml
+    $ bin/deploy --full-build -n app-pg-pointing-at-es --es-ip "$ES_IP" --no-indexing
+
+
+#### Demo Cluster(es-wait) with open postgres port: es-nodes-template.yml and app-pg-template.yml
+    $ export CLUSTER_NAME='encd-dev-open'
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-wait
+
+    ### Output
+    Deploying es-nodes
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-wait
+    Create instance and wait for running state
+    # Deploying Head ES Node(172.31.29.190): encd-dev-open-datamaster
+    ###
+
+    The IP address is the --es-ip used to deploy a frontend.
+    $ export ES_IP='172.31.29.190'
+
+    $ bin/deploy --full-build --cluster-name "$CLUSTER_NAME" --es-ip "$ES_IP" --pg-open
+    
+    ### Output
+    Deploying app-pg
+    $ bin/deploy --full-build --cluster-name encd-dev-open --es-ip 172.31.29.190 --pg-open
+    Create instance and wait for running state
+
+    Deploying Frontend(172.31.25.255): https://encd-dev.demo.encodedcc.org
+    ###
+
+
+#### Demo with postgres and elasticsearch pointing at Demo Cluster: app-template.yml
+    $ export PG_IP='172.31.25.255'
+    $ bin/deploy --full-build -n app-pointing-at-pg-es --cluster-name "$CLUSTER_NAME" --es-ip "$ES_IP" --pg-ip "$PG_IP"
```

Comment:
Read/only node?  Should this be --no-indexing?

Reply:
The app-template doesn't need the --no-indexing flag.  But yes, it does not have the indexing process.  
This is the one we'll be using to scale prod along with the open postgres port.   So current prod deploy will be as it is now with N of these read/only nodes.